tests/test_accounts.py

```diff
@@ -1,11 +1,11 @@
+from binascii import hexlify, unhexlify
+
 import pytest
 
-from binascii import hexlify, unhexlify
-from nano.crypto import private_to_public_key
 from nano.accounts import (
-    address_checksum, public_key_to_xrb_address, xrb_address_to_public_key,
```

Comment:
We can't remove this import from the test - in case there is code that already uses `from nano.accounts import import address_checksum`.

Reply:
added back, sorry